efiction/chapters.py

```diff
@@ -42,10 +43,11 @@ def __file_with_path(dirpath, subdir, filename):
     def __load_chapter_text_into_db(self, chapter_paths: List[dict]):
         """
         Load chapters text from the `stories` files into the chapters table. Uses Windows 1252 if UTF-8 fails.
+        Load chapters text from the `stories` files into the chapters table. Checks for an encoding in the ini file and removes characters that are invalid in that encoding.
```

Comment:
For some reason this ended up on the wrong line (I think in my original PR) - it was meant to replace the text on line 45 which is no longer true

Reply:
Fixed!